nativepython/type_wrappers/string_wrapper.py

```diff
@@ -165,6 +167,116 @@ def convert_method_call(self, context, instance, methodname, args, kwargs):
                         ).cast(self.layoutType)
                     )
                 )
+        elif methodname == "upper":
```

Comment:
might make sense to have a dictionary from method to the runtime function if the pattern is consistent

Reply:
Yes, I agree.  Will do.